pentaho-gwt-widgets/src/org/pentaho/gwt/widgets/themes/public/themes/sapphire/globalSapphire.css

```diff
@@ -4256,13 +4256,14 @@ button.adminButton {
 .dijitToolbar .pc_pageTotal {
   padding: 4px 0px 0px 2px;
 }
-
+/*Overrides Analyzer styles for filter panels*/
 .reportArea #selectionFilterPanel {
-  padding: 1px 3px 8px 3px;
-  background-color: #8ab3c6;
-  border-left: 1px #CCC solid;
-  border-right: 1px #CCC solid;
-  border-bottom: 1px #CCC solid;
+  margin-top: 2px;

```

Comment:
if you use margin on top, probably need decrease padding on the top. What do you think?